test/kitchensink.test.js

```diff
@@ -242,6 +249,31 @@ describe('app', () => {
       const body = resp.body.toString();
       expect(body).not.toContain('<br>');
     });
+
+    test('should add background color', async () => {
+      const resp = await request.get(
+        '/svg/eyJjb2RlIjoiZ3JhcGggVERcbiAgQVtMb3JlbSBpcHN1bSBkb2xvciBzaXQgYW1ldCw8YnIgLz5jb25zZWN0ZXR1ciBhZGlwaXNjaW5nIGVsaXQuXSIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In19?bgColor=123123'
+      );
+      const body = resp.body.toString();
+      expect(body).toMatch(/<svg [^>]* background-color: rgb\(18, 49, 35\);/);
+    });
+
+    test('should add named background color', async () => {
+      const resp = await request.get(
+        '/svg/eyJjb2RlIjoiZ3JhcGggVERcbiAgQVtMb3JlbSBpcHN1bSBkb2xvciBzaXQgYW1ldCw8YnIgLz5jb25zZWN0ZXR1ciBhZGlwaXNjaW5nIGVsaXQuXSIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In19?bgColor=!slategray'
+      );
+      const body = resp.body.toString();
+      expect(body).toMatch(/<svg [^>]* background-color: slategray;/);
+    });
+
+    test('should silently ignore invalid background colors', async () => {
+      const resp = await request.get(
+        '/svg/eyJjb2RlIjoiZ3JhcGggVERcbiAgQVtMb3JlbSBpcHN1bSBkb2xvciBzaXQgYW1ldCw8YnIgLz5jb25zZWN0ZXR1ciBhZGlwaXNjaW5nIGVsaXQuXSIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In19?bgColor=!non-existant-color'
+      );
+      expect(resp.status).toEqual(200);
+      const body = resp.body.toString();
+      expect(body).not.toMatch(/<svg [^>]* background-color:/);
+    });
```

Comment:
Could you please add some tests to ensure the validation (regex) is working?

Reply:
For the tests, even if the regex doesn't work, passing something that is not a color to `background.color` will be ignored, so I am not sure I can do more using the current test layou than the *invalid color* test already present. 

Should I put the regex on an exported function so it can be tested ? What do you have in mind ?

Reply:
The tests could be passing in some invalid input of color, assert that `background-color` is intact.

I think adding the tests to integration test (`test/kitchensink.test.js`) should be sufficient.

Reply:
well, there is already the one testing if passing `bgColor=invalid-color` leaves the `background-color` intact (see *'should silently ignore invalid background colors'*), but I can add another one if you want.

Reply:
I still do not understand. The test above tests that if something is invalid (doesn't pass the regex, or is not a valid color), the `background-color` attribute is absent from the output, meaning there is no `background-color` style at all and thus the `background-color` is intact. What else would you assert ? Or what other test would you do ? Do you mean duplicate the test, this time with something such as `bgColor=javascript:alert("test")` ?

Reply:
Oh, right! I misread the assertion in the test (`should silently ignore invalid background colors)`. 

This is sufficient.